### PR TITLE
Remove bokeh templating from Dark monitor

### LINKED ISSUE #1099: dark monitor: add units to colorbar for dark image view
The colorbar is present, but has no units. 

It may also be helpful to use the image scaling functionality that was recently added to the preview images/thumbnails, in order to bring out more details in the dark image.

### PR DESCRIPTION
**Pull request by bhilbert4**
Replace bokeh templating with straight Bokeh for the dark monitor.
This will resolve the first half of #1141

### CODE REVIEW

**Comment by bhilbert4** [comment]
Getting close. Just need to double check that file date ranges are being propagated correctly (use nirspec as test case), and add date ranges to the histogram plots.

Right now lists of bad pixels are only retrieved from the database if they are short (under 1000 elements long). Retrieving longer lists was noticeably slowing the db query. But the dark monitor is still very slow. I'm not sure why. And the NIRCam page times out or won't load, I'm guessing because it is trying to display 10x the plots an images compared to the other instruments. Not sure yet how to solve this.

**Comment by bhilbert4** [comment]
@mfixstsci this is ready for review!

File: jwql/website/apps/jwql/bokeh_containers.py [hunk 1]
```diff
@@ -570,3 +509,88 @@ def readnoise_monitor_tabs(instrument):
...
+    elif instrument.lower() == 'fgs':
+        full_frame_lists = [
+            [plots['FGS1_FULL'], plots['FGS2_FULL']]
+        ]
+
+    # Next create lists of subarrays. Keep the subarrays in the order in which
+    # they exist in pyiaf, in order to make the page a little more readable.
+    # The dark monitor also populates aperture names using pysiaf.
+    subarrs = [p for p in plots.keys() if p not in FULL_FRAME_APERTURES[instrument.upper()]]
+    siaf = pysiaf.Siaf(instrument.lower())
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/bokeh_containers.py:567]: Nice, love to see people using pysiaf.

File: jwql/website/apps/jwql/bokeh_containers.py [hunk 2]
```diff
@@ -570,3 +509,88 @@ def readnoise_monitor_tabs(instrument):
...
+        row_list = [plots[key] for key in row_keys]
+        subarr_lists.append(row_list)
+
+    # Combine full frame and subarray aperture lists
+    full_list = full_frame_lists + subarr_lists
+
+    # Now create a layout that holds the lists
+    plot_layout = layout(full_list)
+
+    return plot_layout
```
Comment (mfixstsci) [inline jwql/website/apps/jwql/bokeh_containers.py:596]: Very clean and easy to read.

File: environment_python_3_9.yml [hunk 3]
```diff
@@ -62,6 +62,7 @@ dependencies:
     - pysqlite3==0.4.7
     - pyvo==1.4
     - redis
+    - selenium==4.8.0
```
Comment (mfixstsci) [inline environment_python_3_9.yml:65]: @bhilbert4 I've looked twice and didn't notice this package being imported in the changes you made? Do we need to include this dependency, am I not looking hard enough?

**Review by mfixstsci — APPROVED** [review]
@bhilbert4 Thanks for submitting this. Everything looks great, as expected, I just have a couple of quick questions and comments.

File: environment_python_3_9.yml (hunk 3 above)
Comment (bhilbert4) [reply environment_python_3_9.yml:65]: This is required by the PIL in order to produce png files. I already had the PIL in my environment, but not selenium. Maybe the PIL doesn't install it automatically? Do you have the PIL and/or selenium in your env?

Comment (mfixstsci) [reply environment_python_3_9.yml:65]: Cool, I figured it was necessary but I didn't see the import declared in the code base anywhere so I just wanted to double check. Thanks!

**Pull request merged**